nevergrad/benchmark/test_experiments.py

```diff
@@ -25,7 +25,9 @@ def test_experiments_registry(name: str, maker: tp.Callable[[], tp.Iterator[expe
     with datasets.mocked_data():  # mock mlda data that should be downloaded
         check_maker(maker)  # this is to extract the function for reuse if other external packages need it
     if name not in {"realworld_oneshot", "mlda", "mldaas", "realworld", "rocket", "mldakmeans",
-                    "naivemltuning", "seqmltuning", "naiveseqmltuning", "mltuning", "control_problem"}:
+                    "naivemltuning", "seqmltuning", "naiveseqmltuning", "mltuning", "control_problem",
+                    "noisy_control_problem", "para_noisy_control_problem", "para_control_problem", "mujoco",
+                    "noisy_mujoco", "para_noisy_mujoco", "para_mujoco", "mujoco_humanoid"}:
```

Comment:
please simplify this, you can check if the name contains mujoco or mlda or control_problem for instance, that will make it way shorter.